eng/pipelines/templates/jobs/ci.yml

```
@@ -398,6 +398,11 @@ jobs:
         parameters:
           JdkVersion: ${{ parameters.JavaBuildVersion }}
 
+      - task: NodeTool@0
+        inputs:
+          versionSpec: 20.x
+        displayName: Use Node.js 20.x
```

```suggestion
      - task: UseNode@1
        inputs:
          versionSpec: 22.x
        displayName: Use Node.js 22.x
```

Please use latest version of task, now named `UseNode@1`.  Unless you specifically need Node 20, I recommend going straight to Node 22, since it will be supported longer.

---

Thanks. I think Node 22 should work fine. (locally I uses 22)

I will trigger the test PR to double check.